CMakeLists.txt

```diff
@@ -5,7 +5,9 @@
 cmake_minimum_required(VERSION 3.8)
 
 project("Libmultiprocess" CXX)
-set(CMAKE_CXX_STANDARD 17)
+if(NOT DEFINED CMAKE_CXX_STANDARD)
```

Comment:
In commit "build: don't clobber user/superproject c++ version" (bf07b262ba43a000f7605d050de2b6e39fd350e7)

I'm wondering if you think it might be better to do

```
if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
  set(CMAKE_CXX_STANDARD 17
  set(CMAKE_CXX_STANDARD_REQUIRED YES)
endif()
```

to handle the CMAKE_CXX_STANDARD and CMAKE_CXX_STANDARD_REQUIRED variables together instead of separately, to avoid clobbering CMAKE_CXX_STANDARD_REQUIRED, and to be more consistent with parent project detection code below.

Reply:
Done.

Reply:
> Done.

Did you push the correct branch?

Reply:
Heh, whoops. Thanks. Fixed.